### PR TITLE
Update import model for MMDetection Models

### PR DESCRIPTION
**Pull request by rjaincc**
# Description
Updating model import with MMDetection models
(Failed) Run for HF: https://ml.azure.com/experiments/id/88d163bd-e355-47bc-b785-b53b16955805/runs/amusing_pizza_gn3kqh7d1m?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/rupaljain-rs/providers/Microsoft.MachineLearningServices/workspaces/rupaljain-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#
- Please note that this run fails at download model component in the pipeline, as the model already exists in registry. Therefore, in the notebook, we have added a check which allows us to determine if the run should be scheduled (when model is not present in registry) or not (when model is present in registry). 

(Successful) Run for HF: https://ml.azure.com/experiments/id/88d163bd-e355-47bc-b785-b53b16955805/runs/epic_queen_vr70rx6hvx?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/rupaljain-rs/providers/Microsoft.MachineLearningServices/workspaces/rupaljain-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#
Run for MMD: https://ml.azure.com/experiments/id/88d163bd-e355-47bc-b785-b53b16955805/runs/brave_soccer_3rs7g0g66m?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/rupaljain-rs/providers/Microsoft.MachineLearningServices/workspaces/rupaljain-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

# Checklist
- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/python/foundation-models/system/import/import_model_into_registry.ipynb [hunk 1]
```diff
@@ -402,10 +405,17 @@
    "outputs": [],
    "source": [
     "pipeline_object = model_import_pipeline(\n",
-    "    model_id=MODEL_ID, compute=COMPUTE, task_name=TASK_NAME\n",
```
Comment (novaturient95) [inline sdk/python/foundation-models/system/import/import_model_into_registry.ipynb:405]: Should this be reverted to make use of MODEL_ID? `MMDETECTION_MODEL_ID` is defined in below sections

Comment (rjaincc) [reply sdk/python/foundation-models/system/import/import_model_into_registry.ipynb:405]: Fixed in latest

File: sdk/python/foundation-models/system/import/import_model_into_registry.ipynb [hunk 2]
```diff
@@ -26,6 +26,10 @@
     "* text-classification\n",
     "* translation\n",
     "* image-classification\n",
+    "* image-object-detection\n",
+    "* image-instance-segmentation\n",
+    "\n",
+    "For `image-object-detection` and `image-instance-segmentation` models, please refer to [3. Create a pipeline job using components for MMDetection models](#3.-Create-a-pipeline-job-using-components-for-MMDetection-models). For all other tasks, refer to [2. Create a pipeline job using the pipeline component for HuggingFace models](#2.-Create-a-pipeline-job-using-the-pipeline-component-for-HuggingFace-models). Rest of the sections work as is for all the tasks listed above.\n",
```
Comment (gauravrajguru) [inline sdk/python/foundation-models/system/import/import_model_into_registry.ipynb:32]: is this link broken? It's not working for me.

File: sdk/python/foundation-models/system/import/import_model_into_registry.ipynb [hunk 3]
```diff
@@ -44,11 +48,10 @@
...
     "\n",
     "### Outline\n",
-    "* Setup pre-requisites such as compute.\n",
-    "* Pick a model to import.\n",
-    "* Configure the import job.\n",
-    "* Run the fine tuning job.\n",
-    "* Register the fine tuned model. \n",
+    "1. Connect to Azure Machine Learning Workspace\n",
+    "2. Create a pipeline job using the pipeline component for HuggingFace models\n",
+    "3. Create a pipeline job using components for MMDetection models\n",
```
Comment (gauravrajguru) [inline sdk/python/foundation-models/system/import/import_model_into_registry.ipynb:53]: nit: can be 2.1 and 2.2

**Review by gauravrajguru — APPROVED** [review]
nit comments

File: sdk/python/foundation-models/system/import/import_model_into_registry.ipynb (hunk 3 above)
Comment (rjaincc) [reply sdk/python/foundation-models/system/import/import_model_into_registry.ipynb:53]: Moved MMD to a separate notebook per Sasi's suggestion offline.

File: .github/workflows/sdk-foundation-models-system-import-import_model_into_registry_new_model.yml [hunk 4]
```diff
@@ -17,6 +17,8 @@ on:
 env:
   MODEL_ID: ai4bharat/indic-bert
   TASK_NAME: fill-mask
+  MMDETECTION_MODEL_ID: None
+  MMDETECTION_TASK_NAME: image-object-detection
```
Comment (skasturi) [inline .github/workflows/sdk-foundation-models-system-import-import_model_into_registry_new_model.yml:21]: Is this ignored?

Comment (rjaincc) [reply .github/workflows/sdk-foundation-models-system-import-import_model_into_registry_new_model.yml:21]: Yes, it will not run MMDet models.

**Review by novaturient95 — APPROVED** [review]

**Pull request merged**